.github/ISSUE_TEMPLATE/registration-issues.md

```diff
@@ -25,6 +25,10 @@ If all the above looks good you can open an issue.
 
 **Link to the intermediate certificate on the [EU Trusted list](https://eidas.ec.europa.eu/efda/tl-browser/#/screen/home):**
 
+**Debug information**
+* call `curl -k https://debug.openbanking-sandbox.prod.lunar.tech --cert your_certificate --key your_private_key`
```

Comment:
I'm not sure we should mandate the use of `curl`. It would be best if they could call using whatever tool they are currently using, as to best debug why they currently cannot register.